src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessTest.cs

```diff
@@ -79,25 +80,40 @@ public void SetAndCheckBasePriority(ProcessPriorityClass exPriorityClass, int pr
             Assert.Equal(priority, _process.BasePriority);
         }
 
-        [Fact]
+        [Fact, PlatformSpecific(PlatformID.Windows)]
         public void Process_BasePriority()
         {
             ProcessPriorityClass originalPriority = _process.PriorityClass;
             Assert.Equal(ProcessPriorityClass.Normal, originalPriority);
 
-            if (global::Interop.IsWindows)
+            try
             {
-                try
-                {
-                    //SetAndCheckBasePriority(ProcessPriorityClass.RealTime, 24);
-                    SetAndCheckBasePriority(ProcessPriorityClass.High, 13);
-                    SetAndCheckBasePriority(ProcessPriorityClass.Idle, 4);
-                    SetAndCheckBasePriority(ProcessPriorityClass.Normal, 8);
-                }
-                finally
-                {
-                    _process.PriorityClass = originalPriority;
-                }
+                //SetAndCheckBasePriority(ProcessPriorityClass.RealTime, 24);
```

Comment:
From msdn, 

> RealTime priority preempt the threads of all other processes, including operating system processes performing important tasks. Thus, a RealTime priority process that executes for more than a very brief interval can cause disk caches not to flush or cause the mouse to be unresponsive.

I am not sure if we should be doing this in test, as it could potentially hang the machine.